### PR TITLE
Add lymphoma branch ols

### PR DESCRIPTION
**Pull request by matentzn**
@twhetzel sorry I made the PR too hastily and it was at the wrong place.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,19 +17,23 @@ clean:
 
 ontologies: $(ONTFILES)
 
-ontologies/mondo-issue-%.owl:
-	mkdir -p github && mkdir -p github/mondo-issue-$* && rm -rf github/mondo-issue-$*/*
-	cd github/mondo-issue-$* && git clone --depth 1 https://github.com/monarch-initiative/mondo.git -b issue-$* 
-	$(ROBOT) merge -i github/mondo-issue-$*/mondo/src/ontology/mondo-edit.obo --catalog github/mondo-issue-$*/mondo/src/ontology/catalog-v001.xml remove --select ontology reason --reasoner ELK -o $@.tmp.owl && mv $@.tmp.owl $@
+ontologies/mondo-branch-%.owl:
```
Comment (twhetzel) [inline Makefile:20]: curious, why did you change issue to branch? issue was a bit more clear, since branch is ambiguous between github and the ontology branches themselves

Comment (matentzn) [reply Makefile:20]: Because I wanted to accommodate for arbitrary branch names, not just those starting with "issue". Previously I made the assumption that a branch starts with the string "issue", which is mostly, but not always, true.
